chain/rpc/src/client.rs

```diff
@@ -174,7 +174,7 @@ impl RetryPolicy<JsonRpcProviderClientError> for SimpleJsonRpcRetryPolicy {
             return NoRetry;
         }
 
-        debug!("retry queue size is {retry_queue_size}");
+        debug!(size = retry_queue_size, "checking retryable error retry queue size");
```

Comment:
This phrasing is difficult to understand, I don't get the log at all, what is it supposed to mean?

Reply:
ok, rephrased